modules/standard/FileSystem.chpl

```diff
@@ -781,7 +925,15 @@ proc getMode(out error: syserr, name: string): int {
 
    :throws SystemError: Thrown to describe an error if one occurs.
 */
-proc getFileSize(name: string): int throws {
+inline proc getFileSize(name: string): int throws {
```

Comment:
For functions that don't return a string, it would be just fine to call the bytes version from the string one, using a cast, FYI. I am also happy with this approach.